docs/DeveloperGuide.adoc

```diff
@@ -279,51 +279,107 @@ Refer to the guide <<DevOps#, here>>.
+available
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |forgetful librarian |mark a book as loaned out |tell borrowers if the book is available
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |forgetful librarian |mark a book as returned |let borrowers know that the book will now be available
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |librarian |generate a list of overdue books and their borrowers |know which borrower has overdue books
+and which books are overdue
 
-_{More to be added}_
+|`* * *` |librarian |generate a list of currently loaned / available books |do inventory checks
+
+|`* * *` |meticulous librarian |record the movement of books in and out |keep track of available books here
+
+|`* * *` |helpful librarian |assist a borrower to registering in the system |help new borrowers start borrowing books
+
+|`* * *` |librarian |search for certain book by the author |recommend other books of the same author
+
+|`* * *` |librarian |search for certain book by its genre |recommend other books of the same genre
+
+|`* * *` |meticulous librarian |different physical books to have different serial numbers |distinguish between
+books of the same title
+
+|`* * *` |librarian |set the default loan period, renew period and fine amount |customize the app to
+suit my library's policies
+
+|`* *` |librarian |extend a book’s loan |help borrowers to borrow the book for a longer period
+
+|`* *` |lazy librarian |generate and record the fine of overdue books |keep track of overdue fines incurred by
+borrowers
+
+|`* *` |dutiful librarian |record that a fine is paid |keep track of accounting and prevent duplicate payments
+
+|`* *` |librarian |view details of a book |know more information about the book - author, genre, synopsis, etc
+
+|`* *` |careless librarian |be able to undo a command |undo my input mistakes
+
+|`* *` |careless librarian |be able to redo a command |undo my undo commands, in case I need it, without having to
+type out a possibly lengthy command
+
+|`* *` |health conscious, night-working librarian |change the user interface into a night mode |reduce the impact
+of light and glare on my eyes when I am working at night
+
+|`* *` |impatient librarian |have my command inputs returned within 1 sec |serve my customers quickly
+
+|`* *` |forgetful librarian |look at the help section |be reminded of the commands available
+
+|`* *` |helpful librarian |be able to reserve a currently on-loan book |allow borrowers to borrow the book once it
+is returned
+
+|`* *` |librarian |be able to see an image of the book cover |be reminded of the commands available
```

Comment:
Function does not match benefit

Reply:
Thank you for pointing this out, we will amend it.